jekyll/_cci2/ecs-ecr.md

```diff
@@ -81,323 +81,57 @@ AWS_RESOURCE_NAME_PREFIX | Prefix for some required AWS resources. Should corres
-            pip install -r requirements.txt
-```
-
-#### Set Up Caching Steps
-
-Use [`save_cache`]({{ site.baseurl }}/2.0/configuration-reference/#save_cache) and [`restore_cache`]({{ site.baseurl }}/2.0/configuration-reference/#restore_cache)
-to cache the installation of the AWS CLI.
-
-{% raw %}
-
-```yaml
-version: 2
-jobs:
-  # ...
-  deploy:
-    # ...
-    steps:
-      # ...
-      - restore_cache:
-          key: v1-{{ checksum "requirements.txt" }}
-      - run:
-          name: Install the AWS CLI
-          command: |
-            python3 -m venv venv
-            . venv/bin/activate
-            pip install -r requirements.txt
-      - save_cache:
-          key: v1-{{ checksum "requirements.txt" }}
-          paths:
-            - "venv"
-```
-
-{% endraw %}
-
-See the [Caching Dependencies document]({{ site.baseurl }}/2.0/caching/) for more information.
-
-#### Load Docker Image and Set Environment Variables
-
-[Load the Docker image](https://docs.docker.com/engine/reference/commandline/load/) from the workspace,
-then set the following environment variables for convenience:
-
-- the name of the ECR repository
-- the name of the ECS cluster
-- the name of the ECS service
-
-```yaml
-version: 2
-jobs:
-  # ...
-  deploy:
-    # ...
-    steps:
-      # ...
-      - run:
-          name: Load image
-          command: |
-            docker load --input workspace/docker-image/image.tar
-      - run:
-          name: Setup common environment variables
-          command: |
-            echo 'export ECR_REPOSITORY_NAME="${AWS_RESOURCE_NAME_PREFIX}"' >> $BASH_ENV
-            echo 'export ECS_CLUSTER_NAME="${AWS_RESOURCE_NAME_PREFIX}-cluster"' >> $BASH_ENV
-            echo 'export ECS_SERVICE_NAME="${AWS_RESOURCE_NAME_PREFIX}-service"' >> $BASH_ENV
+version: 2.1
+orbs:
+  aws-ecr: circleci/aws-ecr@0.0.2
+  aws-ecs: circleci/aws-ecs@0.0.3
+workflows:
+  build-and-deploy:
+    jobs:
+      - aws-ecr/build_and_push_image:
+          account-url: "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com"
+          repo: "${AWS_RESOURCE_NAME_PREFIX}"
+          region: ${AWS_DEFAULT_REGION}
+          tag: "${CIRCLE_SHA1}"
+      - ...
 ```
 
-**Note:**
-Recall that you must [use `BASH_ENV`](#set-environment-variables)
-to set interpolated environment variables.
-
-#### Push Image to ECR, Deploy, and Test Deployment
-
-[Push the image](https://docs.docker.com/engine/reference/commandline/push/) to ECR,
-then deploy the image
-and validate that the deployment succeeded. Refer to the complete [./deploy.sh](https://github.com/circleci/go-ecs-ecr/blob/master/deploy.sh) script for details.
+### Deploy the new Docker image to an existing AWS ECS service
+The `deploy-service-update` job of the aws-ecs orb creates a new task definition that is based on the current task definition, but with the new Docker image specified in the task definition's container definitions, and deploys the new task definition to the specified ECS service.
```

Comment:
Might be good to add a link to https://circleci.com/orbs/registry/orb/circleci/aws-ecs here

Reply:
Fixed